### PR TITLE
Move CloudBigtableIO.writeToTable() test case from CloudBigtableIOTes…

### PR DESCRIPTION
**Pull request by kevinsi4508**
Move CloudBigtableIO.writeToTable() test case from CloudBigtableIOTest to CloudBigtableIOIntegrationTest.

### CODE REVIEW

File: bigtable-hbase-dataflow/src/test/java/com/google/cloud/bigtable/dataflow/CloudBigtableIOTest.java [hunk 1]
```diff
@@ -81,8 +70,6 @@ public void testInitialize() throws Exception{
     checkRegistry(Put.class);
```
Comment (sduskis) [inline bigtable-hbase-dataflow/src/test/java/com/google/cloud/bigtable/dataflow/CloudBigtableIOTest.java]: Can you please move this into the Integration test?  That seems like a better fit.

Comment (kevinsi4508) [inline bigtable-hbase-dataflow/src/test/java/com/google/cloud/bigtable/dataflow/CloudBigtableIOTest.java]: Coder registry is mocked. I think we should keep it in unit testing.
"when(underTest.getCoderRegistry()).thenReturn(registry);"

Comment (sduskis) [inline bigtable-hbase-dataflow/src/test/java/com/google/cloud/bigtable/dataflow/CloudBigtableIOTest.java]: ok

**Comment by sduskis** [comment]
LGTM

**Pull request merged**